src/cluster.c

```diff
@@ -2250,8 +2322,9 @@ int clusterProcessPacket(clusterLink *link) {
                 /* If the reply has a non matching node ID we
                  * disconnect this node and set it as not having an associated
                  * address. */
-                serverLog(LL_DEBUG,"PONG contains mismatching sender ID. About node %.40s added %d ms ago, having flags %d",
+                serverLog(LL_DEBUG,"PONG contains mismatching sender ID. About node %.40s %s added %d ms ago, having flags %d",
```

Comment:
```suggestion
                serverLog(LL_DEBUG,"PONG contains mismatching sender ID. About node %.40s (%s) added %d ms ago, having flags %d",
```
Maybe something like this so it's clear that there is an empty name there? Otherwise it'll look like there is an awkward 2 spaces there. Not a strong opinion here.

Reply:
Yes I agree that would be a better formatting, I have changed to this now. Thanks you